pacman/README.md

```diff
@@ -0,0 +1,155 @@
+source activate <env_name>
+```
+
+From now, it is assumed that <env_name> is activated. 
+
+ - Install dependencies.
+```bash
+pip install -U gym numpy stopit argparse pynput pillow
+```
+
+ - Install and use pycodestyle to check PEP8 compliance of Python scripts.
+```bash
+pip install -U pycodestyle
+pycodestyle <yourscript>.py
+```
+
+ - Install and use autopep8 to rewrite Python scripts into PEP8 format.
+```bash
+pip install -U autopep8
+autopep8 --in-place --aggressive --aggressive <yourscript>.py
+```
+
+/!\ Check your file again with pycodestyle.
+
+# Usage
+
+## General
+
+ - Display the command-line help section
+```bash
+python run.py -h
+```
+
+  - Interactive game
+```bash
+python run.py --agentfile humanagent.py
+```
+:warning: While it works directly on Linux, root privileges are needed for keyboard access in MacOS. Not tested for Windows.
+
+## Search (1/3)
+
+ - Launches a game with a random agent in default map mediumClassic:
+```bash
+python run.py
+```
+
+ - Launches a game with your Pacman agent class, named PacmanAgent located in `youragentmodule.py`, and with the maze originalClassic (see layout folder).
+```bash
+python run.py --layout originalClassic --agentfile youragentmodule.py
+```
+ - Launches a game with the human agent (that is, an interactive game).
+```bash
+python run.py --layout originalClassic --agentfile humanagent.py
+```
+
+- Same configuration as above, but enable the call to `registerInitialState` method.
+```bash
+python run.py --layout mediumClassic --agentfile youragentmodule.py --registerinitialstate
+```
+
+
+See the help section of the command line for more information about the parameters.
+
+# Scoring
+
+### Search (1/3)
+
+The score function is computed as follows:
+
+`score = -#time steps + 10*#number of food dots eaten + (500 when finished)`
+
+# Instructions
+
+## Introduction 
+
+![Pacman maze example](https://github.com/glouppe/info8006-introduction-to-ai/blob/pacman_project/pacman/pacman_game.png "Pacman maze")
+
+In this classical Pacman game, the player navigates Pacman through a maze filled of food dots. The goal is to collect them all in a minimum amount of time (Part 1/3). Ghosts, which are Pacman's enemies, may also be part of the game. In this configuration, Pacman needs to avoid them while collecting all the food dots (Part 2/3). To help him, Pacman may have access to capsules in the maze, giving him the ability to beat Ghosts, resetting them in their initial positions, in a short amount of time. The game ends if either Pacman have collected all the dots (winning end) or Pacman collides with one of the Ghosts (losing end). Cherry on the cake, some walls on the game may have a particular characteristic. Indeed, they can randomly be crossed or not over time (Part 3/3).
+
+The project is splitted into three parts. Your task is to design a family of searching agents for each part of the project (details are provided in the three next sections below). 
```

Comment:
"Your task is to design a family of searching agents for each part of the project"

rather say "design an intelligent agent for each part of the project"